<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageInventory</class>
 <widget class="QDialog" name="ManageInventory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qu·∫£n L√Ω H√†ng Trong Kho</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resource.qrc">
    <normaloff>:/icons/bag.png</normaloff>:/icons/bag.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>QU·∫¢N L√ù H√ÄNG TRONG KHO</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutSearch">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="labelFilter">
       <property name="text">
        <string>L·ªçc theo:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbFilter">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>45</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>To√†n b·ªô</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ƒê·ªì ƒÉn</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Th·ª©c u·ªëng</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ƒê·ªì gia d·ª•ng</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtSearch">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>45</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>T√¨m ki·∫øm theo t√™n ho·∫∑c m√£ s·∫£n ph·∫©m...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRefresh">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>45</height>
        </size>
       </property>
       <property name="text">
        <string>L√†m m·ªõi</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblTotal">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>T·ªïng s·ªë s·∫£n ph·∫©m: 0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblInstruction">
     <property name="styleSheet">
      <string notr="true">color: #6C757D; font-style: italic;</string>
     </property>
     <property name="text">
      <string>üí° Nh·∫•p ƒë√∫p v√†o s·∫£n ph·∫©m ƒë·ªÉ ch·ªânh s·ª≠a th√¥ng tin</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableProducts">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>50</number>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutButtons">
     <item>
      <widget class="QPushButton" name="btnDelete">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #DC3545;
    color: white;
    border: none;
    border-radius: 20px;
    font-weight: 600;
}

QPushButton:hover {
    background-color: #C82333;
}

QPushButton:pressed {
    background-color: #BD2130;
}</string>
       </property>
       <property name="text">
        <string>X√≥a S·∫£n Ph·∫©m</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::StandardButton::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ManageInventory</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>